### ISSUE TITLE
[Feature Request] Quick Save

### ISSUE THREAD

**Issue by shenzhiming88** [NickeManarin/ScreenToGif#830]
Hello!
Creating GIF motion pictures may be more for sharing on the network, but the current Save As process is a bit tedious. Therefore, I would like to add a "Quick Save" button that can do the following

- Save the file to a specified directory with one click, and that directory can be customized

- Copy the image to the clipboard while saving it for easy sending

- Ability to customize the configuration of the save

Thanks!

**Comment by tony** [comment]
I agree, and in addition it would also be nice for Quick save to include template/formatting of filenames (#282, #837) to expedite saving (and assure filenames don't overlap)
